src/components/ChartContextMenu.svelte

```diff
@@ -65,18 +65,24 @@
       : 'main_1pct';
 
   const REDASH_PROBE_COMPARISON_URL =
-    'https://sql.telemetry.mozilla.org/queries/82247/source?';
+    'https://sql.telemetry.mozilla.org/dashboard/test_38?';
 
   const getComparisonViewinSTMO = (clicked, hovered) => {
     const queryParams = new URLSearchParams({
       p_Table: `telemetry.${table}`,
       p_Probe: getTelemetryPath(),
-      p_OS: $store.productDimensions.os,
       'p_Build 1': clicked,
       'p_Build 2': hovered,
       'p_Start Date': dateFormatter(getDateFromPoint(clicked)),
       'p_Start Date 2': dateFormatter(getDateFromPoint(hovered)),
     });
+    // do not add OS filter to the query if 'All OSes' is selected
+    queryParams.append(
```

Comment:
I don't think you need to use append here. Maybe just put it at the end of the object definition above with the comment?